Production_scripts/Lookup-01b-Postcode-tests.R

```diff
@@ -19,5 +19,12 @@ old_tests <- produce_slf_postcode_tests(haven::read_sav(get_slf_postcode_path(up
 # create tests
 comparison <- produce_test_comparison(old_tests, new_tests)
 
+# Save output as zsav for now
+# Eventually change this to rds when we have more R scripts
+haven::write_sav(comparison,
+                 path = glue::glue("/conf/hscdiip/SLF_Extracts/Lookups/source_postcode_lookup_{latest_update()}_tests.zsav",
```

Comment:
Use `get_slf_dir` function here (and in others).

Reply:
I have updated the zsav file path but also added in the option to save tests as an xlsx workbook. This may need some changes but the idea is to load the workbook, add a new sheet, write data and save out.